.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,11 @@
+blank_issues_enabled: false
```

Comment:
Are you ok with this? Personally I would prefer if we started moving towards discussions for feature requests

Reply:
does this mean people can't open an issue with no description? (could they use a custom description that doesn't fit our template still?)

Reply:
yeah, it means people would have to follow the template when opening issues. can remove this if needed

Reply:
ideally we could just add a new template for other issues which you can type anything in

Reply:
added misc issue template, hopefully that should be enough